app/containers/ChangeLocale/index.js

```diff
@@ -20,6 +20,8 @@ export class ChangeLocale extends React.Component {
 
   /* eslint react/no-did-mount-set-state: 0 */
   componentWillMount() {
+    ChangeLocale.changeLanguage = ChangeLocale.changeLanguage.bind(this);
+
```

Comment:
Do NOT used bind(this). Make changeLanguage arrow function instead.

https://medium.com/@machnicki/handle-events-in-react-with-arrow-functions-ede88184bbb

Reply:
i have seen it, but i haven't found a way to save context inside of STATIC method (also with help of arrow function)